src/components/radio-button/radio-button.config.js

```diff
@@ -118,19 +110,5 @@ module.exports = {
         items: items(),
       },
     },
-    {
-      name: 'vertical-top',
```

Comment:
Did we want to remove this variant?

Reply:
Yes both of these were for v10 only but after speaking with @jeanservaas we don't want to offer these variants as they are not correct design wise.

Reply:
Got it - Thank you for clarifying @aledavila!